objectivec/Tests/GPBWellKnownTypesTest.m

```diff
@@ -99,4 +101,54 @@ - (void)testDuration {
   [duration2 release];
 }
 
+- (void)testAnyPackFromUnpackTo {
+  TestAllTypes *from = [TestAllTypes message];
+  [self setAllFields:from repeatedCount:1];
+  NSData *data = from.data;
+
+  // Test packFrom.
+  GPBAny *any = [[GPBAny alloc] init];
```

Comment:
because you haven't called autorelease on it.

Iniside of the call [TestAllTypes message] the return value is autoreleased.